controller/cli/main.py

```diff
@@ -17,9 +17,11 @@
 
 from controller.api.v10 import rest
 from controller.service import api
-
+from controller.utils import logger
 
 def main():
     app = Flask(__name__)
     app.register_blueprint(rest)
+    logger.configure_logging()
```

Comment:
Before adding this, logging wasn't working. Is this the best location for this?

Reply:
I guess it's ok since is one of the first things we want to do (logging).